push-notifications/ios/Plugin/PushNotificationsPlugin.swift

```diff
@@ -47,6 +47,16 @@ public class PushNotificationsPlugin: CAPPlugin {
         call.resolve()
     }
 
+    /**
+     * Unregister for remote notifications
+     */
+    @objc func unregister(_ call: CAPPluginCall) {
+        DispatchQueue.main.async {
+            UIApplication.shared.unregisterForRemoteNotifications()
+        }
+        call.resolve()
```

Comment:
The resolve should be inside the `DispatchQueue.main.async` block

Reply:
Sure, makes sense. Should `register`'s `resolve` also be in that block? I just copied the pattern.

Reply:
yeah, it should, good catch!